src/Core/Managed/Shared/Extensibility/Implementation/OperationTelemetry.cs

```diff
@@ -10,6 +10,8 @@
     /// </summary>
     public abstract class OperationTelemetry : ITelemetry, ISupportMetrics, ISupportProperties
     {
+        internal const string TelemetryName = "Operation";
```

Comment:
It's used for the event name in ProcessOperationStart/Stop. I don't distinguish between Request/DependencyTelemetry - they're both "Operation". I followed the pattern in ```Process(ITelemetry)```. The alternative was to hard-code the string.